### PR TITLE
Update requirements.txt and drop support for Django1.7 and Django1.6

### PR DESCRIPTION
**Pull request by KevinEtienne**
.. and add support for `Django` `1.9`

### CODE REVIEW

**Comment by KevinEtienne** [comment]
Looks like [`django-mptt>=0.8`](https://github.com/django-mptt/django-mptt/releases/tag/0.8.0) has dropped support for `django<1.8`.

**Comment by KevinEtienne** [comment]
I can drop support for `django1.7` and `django1.6`, does that seem reasonable @Ian-Foote @adam-thomas?

**Comment by adam-thomas** [comment]
Yeah, old projects can just use past versions of this library.  Works for me :)

**Comment by Ian-Foote** [comment]
Might as well add support for 1.9 while you're at it.

**Comment by KevinEtienne** [comment]
Good idea, thanks

**Comment by KevinEtienne** [comment]
Updated title and description

File: tests/testcases/test_admin_request.py [hunk 1]
```diff
@@ -40,6 +44,15 @@ def test_get_admin_change_page(self):
         expected = '/admin/tests/user/{}/'.format(user.pk)
         self.client.get.assert_called_with(expected)
 
+    @skipUnless(is_django_gte_19(), "Django 1.9 has updated the admin url.")
+    def test_get_admin_change_page_django_gte_19(self):
```
Comment (Ian-Foote) [inline tests/testcases/test_admin_request.py]: I'd prefer to use the nice name for the new test and rename the test of (now) legacy functionality.

File: tests/testcases/utils.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import django
+
+
+def is_django_gte_19():
+    return django.VERSION >= (1, 9)
```
Comment (Ian-Foote) [inline tests/testcases/utils.py]: This doesn't need to be a function:
```python
DJANGO_GTE_19 = django.version >= (1, 9)
```

Comment (meshy) [inline tests/testcases/utils.py]: Also, this should be in the [`compat`](https://github.com/incuna/incuna-test-utils/blob/master/incuna_test_utils/compat.py) module.

Comment (Ian-Foote) [inline tests/testcases/utils.py]: Talking about `compat` - can we drop some of that module in this PR?

**Comment by KevinEtienne** [comment]
Updated with last comments

**Comment by adam-thomas** [comment]
:+1:

**Pull request merged**